content/en/docs/concepts/instrumentation-scope/_index.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This file should be named just `index.md`, without the leading `_` -- otherwise it declares this as a section, which it is not.

Reply:
oops ... thanks!

Reply:
Here's a fix:

- #3398

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/3399 ... did the same mistake twice

Reply:
Better:

- #3399